src/functions-and-arrays.js

```diff
@@ -1,41 +1,141 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
+function maxOfTwoNumbers(num1, num2) {
+
```

Comment:
here we can directly do

`return Math.max(num1, num2)`

Math is a javascript object with functions, max is one of those functions

here is the documentation

[math max](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max)